art/estimators/classification/pytorch.py

```diff
@@ -510,17 +511,30 @@ def loss_gradient(self, x: np.ndarray, y: np.ndarray, **kwargs) -> np.ndarray:
         import torch  # lgtm [py/repeated-import]
 
         # Apply preprocessing
-        x_preprocessed, y_preprocessed = self._apply_preprocessing(x, y, fit=False)
+        if self.all_framework_preprocessing:
+            if isinstance(x, torch.Tensor):
```

Comment:
Line 515-520: I think the functions `torch.tensor` and `torch.fromnumpy` have similar functionality, so we don't need an `if..else` condition here.

Reply:
Nice, I was not aware of this, fixed with 3fc9975df395814b6cb9f2d7bedecbbf336a44e0